### ISSUE TITLE
bug: Change start page, Custom playback speed, Hide action bar component, Spoof app version and SponsorBlock failed on YTMusic v6.43

### ISSUE THREAD

**Issue by Raven-621** [anddea/revanced-patches#78]
### Type

Error while patching

### Tools used

rvx-builder

### Application

Youtube Music v6.43.52

### Bug description

Change start page, Custom playback speed, Hide action bar component, Spoof app version and SponsorBlock patches failed on YTMusic v6.43.
Also, since it seems that more and more patches are being broken as YT Music updates, i recommend to make the limit of compatible YTM version to be patchable at v6.42.54.

### Error logs

```shell
[builder] INFO: Decoding resources
[builder] INFO: Executing patches
[builder] INFO: Amoled succeeded
[builder] INFO: Background play succeeded
[builder] INFO: Bitrate default value succeeded
[builder] INFO: Certificate spoof succeeded
[builder] SEVERE: Change start page failed: app.revanced.patcher.patch.PatchException: 'Change start page' depends on 'IntentHookPatch' that raised an exception: app.revanced.patcher.patch.PatchException: Failed to resolve FullStackTraceActivityFingerprint at app.revanced.util.BytecodeUtilsKt.getException(BytecodeUtils.kt:27) at app.revanced.patches.music.utils.intenthook.IntentHookPatch.execute(IntentHookPatch.kt:31) at app.revanced.patches.music.utils.intenthook.IntentHookPatch.execute(IntentHookPatch.kt:14) at app.revanced.patcher.Patcher$apply$1.invokeSuspend$executePatch(Patcher.kt:174) at app.revanced.patcher.Patcher$apply$1.invokeSuspend$executePatch(Patcher.kt:156) at app.revanced.patcher.Patcher$apply$1.invokeSuspend(Patcher.kt:203) at app.revanced.patcher.Patcher$apply$1.invoke(Patcher.kt) at app.revanced.patcher.Patcher$apply$1.invoke(Patcher.kt) at kotlinx.coroutines.flow.SafeFlow.collectSafely(Builders.kt:61) at kotlinx.coroutines.flow.AbstractFlow.collect(Flow.kt:230) at app.revanced.cli.command.PatchCommand$run$4$patcherResult$1$1.invokeSuspend(PatchCommand.kt:304) at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108) at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:280) at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85) at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59) at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source) at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38) at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source) at app.revanced.cli.command.PatchCommand.run(PatchCommand.kt:303) at picocli.CommandLine.executeUserObject(CommandLine.java:2026) at picocli.CommandLine.access$1500(CommandLine.java:148) at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461) at picocli.CommandLine$RunLast.handle(CommandLine.java:2453) at picocli.CommandLine$RunLast.handle(CommandLine.java:2415) at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273) at picocli.CommandLine$RunLast.execute(CommandLine.java:2417) at picocli.CommandLine.execute(CommandLine.java:2170) at app.revanced.cli.command.MainCommandKt.main(MainCommand.kt:12) at app.revanced.patcher.Patcher$apply$1.invokeSuspend$executePatch(Patcher.kt:161) at app.revanced.patcher.Patcher$apply$1.invokeSuspend(Patcher.kt:203) at app.revanced.patcher.Patcher$apply$1.invoke(Patcher.kt) at app.revanced.patcher.Patcher$apply$1.invoke(Patcher.kt) at kotlinx.coroutines.flow.SafeFlow.collectSafely(Builders.kt:61) at kotlinx.coroutines.flow.AbstractFlow.collect(Flow.kt:230) at app.revanced.cli.command.PatchCommand$run$4$patcherResult$1$1.invokeSuspend(PatchCommand.kt:304) at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108) at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:280) at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85) at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59) at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source) at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38) at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source) at app.revanced.cli.command.PatchCommand.run(PatchCommand.kt:303) at picocli.CommandLine.executeUserObject(CommandLine.java:2026) at picocli.CommandLine.access$1500(CommandLine.java:148) at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461) at picocli.CommandLine$RunLast.handle(CommandLine.java:2453) at picocli.CommandLine$RunLast.handle(CommandLine.java:2415) at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273) at picocli.CommandLine$RunLast.execute(CommandLine.java:2417) at picocli.CommandLine.execute(CommandLine.java:2170) at app.revanced.cli.command.MainCommandKt.main(MainCommand.kt:12)
[builder] SEVERE: Custom playback speed failed: app.revanced.patcher.patch.PatchException: 'Custom playback speed' depends on 'IntentHookPatch' that raised an exception: app.revanced.patcher.patch.PatchException: 'IntentHookPatch' did not succeed previously at app.revanced.patcher.Patcher$apply$1.invokeSuspend$executePatch(Patcher.kt:150) at app.revanced.patcher.Patcher$apply$1.invokeSuspend$executePatch(Patcher.kt:156) at app.revanced.patcher.Patcher$apply$1.invokeSuspend(Patcher.kt:203) at app.revanced.patcher.Patcher$apply$1.invoke(Patcher.kt) at app.revanced.patcher.Patcher$apply$1.invoke(Patcher.kt) at kotlinx.coroutines.flow.SafeFlow.collectSafely(Builders.kt:61) at kotlinx.coroutines.flow.AbstractFlow.collect(Flow.kt:230) at app.revanced.cli.command.PatchCommand$run$4$patcherResult$1$1.invokeSuspend(PatchCommand.kt:304) at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108) at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:280) at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85) at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59) at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source) at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38) at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source) at app.revanced.cli.command.PatchCommand.run(PatchCommand.kt:303) at picocli.CommandLine.executeUserObject(CommandLine.java:2026) at picocli.CommandLine.access$1500(CommandLine.java:148) at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461) at picocli.CommandLine$RunLast.handle(CommandLine.java:2453) at picocli.CommandLine$RunLast.handle(CommandLine.java:2415) at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273) at picocli.CommandLine$RunLast.execute(CommandLine.java:2417) at picocli.CommandLine.execute(CommandLine.java:2170) at app.revanced.cli.command.MainCommandKt.main(MainCommand.kt:12) at app.revanced.patcher.Patcher$apply$1.invokeSuspend$executePatch(Patcher.kt:161) at app.revanced.patcher.Patcher$apply$1.invokeSuspend(Patcher.kt:203) at app.revanced.patcher.Patcher$apply$1.invoke(Patcher.kt) at app.revanced.patcher.Patcher$apply$1.invoke(Patcher.kt) at kotlinx.coroutines.flow.SafeFlow.collectSafely(Builders.kt:61) at kotlinx.coroutines.flow.AbstractFlow.collect(Flow.kt:230) at app.revanced.cli.command.PatchCommand$run$4$patcherResult$1$1.invokeSuspend(PatchCommand.kt:304) at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108) at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:280) at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85) at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59) at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source) at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38) at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source) at app.revanced.cli.command.PatchCommand.run(PatchCommand.kt:303) at picocli.CommandLine.executeUserObject(CommandLine.java:2026) at picocli.CommandLine.access$1500(CommandLine.java:148) at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461) at picocli.CommandLine$RunLast.handle(CommandLine.java:2453) at picocli.CommandLine$RunLast.handle(CommandLine.java:2415) at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273) at picocli.CommandLine$RunLast.execute(CommandLine.java:2417) at picocli.CommandLine.execute(CommandLine.java:2170) at app.revanced.cli.command.MainCommandKt.main(MainCommand.kt:12)
[builder] INFO: Disable auto captions succeeded
[builder] INFO: Disable overlay filter succeeded
[builder] INFO: Enable black navigation bar succeeded
[builder] INFO: Enable compact dialog succeeded
[builder] INFO: Enable custom filter succeeded
[builder] INFO: Enable force minimized player succeeded
[builder] INFO: Enable landscape mode succeeded
[builder] INFO: Enable minimized playback succeeded
[builder] INFO: Enable old style library shelf succeeded
[builder] INFO: Enable opus codec succeeded
[builder] INFO: Enable playback speed succeeded
[builder] INFO: Exclusive audio playback succeeded
[builder] INFO: Hide "New" button succeeded
[builder] INFO: Hide account menu succeeded
[builder] SEVERE: Hide action bar component failed: app.revanced.patcher.patch.PatchException: 'Hide action bar component' depends on 'IntentHookPatch' that raised an exception: app.revanced.patcher.patch.PatchException: 'IntentHookPatch' did not succeed previously at app.revanced.patcher.Patcher$apply$1.invokeSuspend$executePatch(Patcher.kt:150) at app.revanced.patcher.Patcher$apply$1.invokeSuspend$executePatch(Patcher.kt:156) at app.revanced.patcher.Patcher$apply$1.invokeSuspend(Patcher.kt:203) at app.revanced.patcher.Patcher$apply$1.invoke(Patcher.kt) at app.revanced.patcher.Patcher$apply$1.invoke(Patcher.kt) at kotlinx.coroutines.flow.SafeFlow.collectSafely(Builders.kt:61) at kotlinx.coroutines.flow.AbstractFlow.collect(Flow.kt:230) at app.revanced.cli.command.PatchCommand$run$4$patcherResult$1$1.invokeSuspend(PatchCommand.kt:304) at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108) at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:280) at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85) at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59) at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source) at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38) at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source) at app.revanced.cli.command.PatchCommand.run(PatchCommand.kt:303) at picocli.CommandLine.executeUserObject(CommandLine.java:2026) at picocli.CommandLine.access$1500(CommandLine.java:148) at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461) at picocli.CommandLine$RunLast.handle(CommandLine.java:2453) at picocli.CommandLine$RunLast.handle(CommandLine.java:2415) at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273) at picocli.CommandLine$RunLast.execute(CommandLine.java:2417) at picocli.CommandLine.execute(CommandLine.java:2170) at app.revanced.cli.command.MainCommandKt.main(MainCommand.kt:12) at app.revanced.patcher.Patcher$apply$1.invokeSuspend$executePatch(Patcher.kt:161) at app.revanced.patcher.Patcher$apply$1.invokeSuspend(Patcher.kt:203) at app.revanced.patcher.Patcher$apply$1.invoke(Patcher.kt) at app.revanced.patcher.Patcher$apply$1.invoke(Patcher.kt) at kotlinx.coroutines.flow.SafeFlow.collectSafely(Builders.kt:61) at kotlinx.coroutines.flow.AbstractFlow.collect(Flow.kt:230) at app.revanced.cli.command.PatchCommand$run$4$patcherResult$1$1.invokeSuspend(PatchCommand.kt:304) at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108) at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:280) at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85) at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59) at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source) at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38) at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source) at app.revanced.cli.command.PatchCommand.run(PatchCommand.kt:303) at picocli.CommandLine.executeUserObject(CommandLine.java:2026) at picocli.CommandLine.access$1500(CommandLine.java:148) at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461) at picocli.CommandLine$RunLast.handle(CommandLine.java:2453) at picocli.CommandLine$RunLast.handle(CommandLine.java:2415) at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273) at picocli.CommandLine$RunLast.execute(CommandLine.java:2417) at picocli.CommandLine.execute(CommandLine.java:2170) at app.revanced.cli.command.MainCommandKt.main(MainCommand.kt:12)
[builder] INFO: Hide button shelf succeeded
[builder] INFO: Hide carousel shelf succeeded
[builder] INFO: Hide cast button succeeded
[builder] INFO: Hide category bar succeeded
[builder] INFO: Hide channel guidelines succeeded
[builder] INFO: Hide double tap overlay filter succeeded
[builder] INFO: Hide emoji picker and time stamp succeeded
[builder] INFO: Hide flyout panel succeeded
[builder] INFO: Hide fullscreen share button succeeded
[builder] INFO: Hide general ads succeeded
[builder] INFO: Hide get premium succeeded
[builder] INFO: Hide handle succeeded
[builder] INFO: Hide history button succeeded
[builder] INFO: Hide navigation bar component succeeded
[builder] INFO: Hide player overlay filter succeeded
[builder] INFO: Hide playlist cards succeeded
[builder] INFO: Hide sample shelf succeeded
[builder] INFO: Hide tap to update button succeeded
[builder] INFO: Hide taste builder succeeded
[builder] INFO: Hide terms container succeeded
[builder] INFO: Hide tooltip content succeeded
[builder] INFO: Hide voice search button succeeded
[builder] INFO: Remember playback speed succeeded
[builder] INFO: Remember repeat state succeeded
[builder] INFO: Remember shuffle state succeeded
[builder] INFO: Remember video quality succeeded
[builder] INFO: Remove viewer discretion dialog succeeded
[builder] INFO: Replace dismiss queue succeeded
[builder] INFO: Return YouTube Dislike succeeded
[builder] INFO: Sanitize sharing links succeeded
[builder] SEVERE: SponsorBlock failed: app.revanced.patcher.patch.PatchException: 'SponsorBlock' depends on 'IntentHookPatch' that raised an exception: app.revanced.patcher.patch.PatchException: 'IntentHookPatch' did not succeed previously at app.revanced.patcher.Patcher$apply$1.invokeSuspend$executePatch(Patcher.kt:150) at app.revanced.patcher.Patcher$apply$1.invokeSuspend$executePatch(Patcher.kt:156) at app.revanced.patcher.Patcher$apply$1.invokeSuspend(Patcher.kt:203) at app.revanced.patcher.Patcher$apply$1.invoke(Patcher.kt) at app.revanced.patcher.Patcher$apply$1.invoke(Patcher.kt) at kotlinx.coroutines.flow.SafeFlow.collectSafely(Builders.kt:61) at kotlinx.coroutines.flow.AbstractFlow.collect(Flow.kt:230) at app.revanced.cli.command.PatchCommand$run$4$patcherResult$1$1.invokeSuspend(PatchCommand.kt:304) at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108) at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:280) at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85) at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59) at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source) at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38) at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source) at app.revanced.cli.command.PatchCommand.run(PatchCommand.kt:303) at picocli.CommandLine.executeUserObject(CommandLine.java:2026) at picocli.CommandLine.access$1500(CommandLine.java:148) at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461) at picocli.CommandLine$RunLast.handle(CommandLine.java:2453) at picocli.CommandLine$RunLast.handle(CommandLine.java:2415) at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273) at picocli.CommandLine$RunLast.execute(CommandLine.java:2417) at picocli.CommandLine.execute(CommandLine.java:2170) at app.revanced.cli.command.MainCommandKt.main(MainCommand.kt:12) at app.revanced.patcher.Patcher$apply$1.invokeSuspend$executePatch(Patcher.kt:161) at app.revanced.patcher.Patcher$apply$1.invokeSuspend(Patcher.kt:203) at app.revanced.patcher.Patcher$apply$1.invoke(Patcher.kt) at app.revanced.patcher.Patcher$apply$1.invoke(Patcher.kt) at kotlinx.coroutines.flow.SafeFlow.collectSafely(Builders.kt:61) at kotlinx.coroutines.flow.AbstractFlow.collect(Flow.kt:230) at app.revanced.cli.command.PatchCommand$run$4$patcherResult$1$1.invokeSuspend(PatchCommand.kt:304) at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108) at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:280) at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85) at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59) at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source) at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38) at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source) at app.revanced.cli.command.PatchCommand.run(PatchCommand.kt:303) at picocli.CommandLine.executeUserObject(CommandLine.java:2026) at picocli.CommandLine.access$1500(CommandLine.java:148) at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461) at picocli.CommandLine$RunLast.handle(CommandLine.java:2453) at picocli.CommandLine$RunLast.handle(CommandLine.java:2415) at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273) at picocli.CommandLine$RunLast.execute(CommandLine.java:2417) at picocli.CommandLine.execute(CommandLine.java:2170) at app.revanced.cli.command.MainCommandKt.main(MainCommand.kt:12)
[builder] SEVERE: Spoof app version failed: app.revanced.patcher.patch.PatchException: 'Spoof app version' depends on 'IntentHookPatch' that raised an exception: app.revanced.patcher.patch.PatchException: 'IntentHookPatch' did not succeed previously at app.revanced.patcher.Patcher$apply$1.invokeSuspend$executePatch(Patcher.kt:150) at app.revanced.patcher.Patcher$apply$1.invokeSuspend$executePatch(Patcher.kt:156) at app.revanced.patcher.Patcher$apply$1.invokeSuspend(Patcher.kt:203) at app.revanced.patcher.Patcher$apply$1.invoke(Patcher.kt) at app.revanced.patcher.Patcher$apply$1.invoke(Patcher.kt) at kotlinx.coroutines.flow.SafeFlow.collectSafely(Builders.kt:61) at kotlinx.coroutines.flow.AbstractFlow.collect(Flow.kt:230) at app.revanced.cli.command.PatchCommand$run$4$patcherResult$1$1.invokeSuspend(PatchCommand.kt:304) at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108) at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:280) at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85) at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59) at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source) at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38) at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source) at app.revanced.cli.command.PatchCommand.run(PatchCommand.kt:303) at picocli.CommandLine.executeUserObject(CommandLine.java:2026) at picocli.CommandLine.access$1500(CommandLine.java:148) at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461) at picocli.CommandLine$RunLast.handle(CommandLine.java:2453) at picocli.CommandLine$RunLast.handle(CommandLine.java:2415) at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273) at picocli.CommandLine$RunLast.execute(CommandLine.java:2417) at picocli.CommandLine.execute(CommandLine.java:2170) at app.revanced.cli.command.MainCommandKt.main(MainCommand.kt:12) at app.revanced.patcher.Patcher$apply$1.invokeSuspend$executePatch(Patcher.kt:161) at app.revanced.patcher.Patcher$apply$1.invokeSuspend(Patcher.kt:203) at app.revanced.patcher.Patcher$apply$1.invoke(Patcher.kt) at app.revanced.patcher.Patcher$apply$1.invoke(Patcher.kt) at kotlinx.coroutines.flow.SafeFlow.collectSafely(Builders.kt:61) at kotlinx.coroutines.flow.AbstractFlow.collect(Flow.kt:230) at app.revanced.cli.command.PatchCommand$run$4$patcherResult$1$1.invokeSuspend(PatchCommand.kt:304) at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33) at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108) at kotlinx.coroutines.EventLoopImplBase.processNextEvent(EventLoop.common.kt:280) at kotlinx.coroutines.BlockingCoroutine.joinBlocking(Builders.kt:85) at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking(Builders.kt:59) at kotlinx.coroutines.BuildersKt.runBlocking(Unknown Source) at kotlinx.coroutines.BuildersKt__BuildersKt.runBlocking$default(Builders.kt:38) at kotlinx.coroutines.BuildersKt.runBlocking$default(Unknown Source) at app.revanced.cli.command.PatchCommand.run(PatchCommand.kt:303) at picocli.CommandLine.executeUserObject(CommandLine.java:2026) at picocli.CommandLine.access$1500(CommandLine.java:148) at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2461) at picocli.CommandLine$RunLast.handle(CommandLine.java:2453) at picocli.CommandLine$RunLast.handle(CommandLine.java:2415) at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2273) at picocli.CommandLine$RunLast.execute(CommandLine.java:2417) at picocli.CommandLine.execute(CommandLine.java:2170) at app.revanced.cli.command.MainCommandKt.main(MainCommand.kt:12)
[builder] INFO: Translations succeeded
[builder] INFO: Settings succeeded
[builder] INFO: Compiling patched dex files
[builder] INFO: Compiled classes.dex
[builder] INFO: Compiled classes2.dex
[builder] INFO: Compiled classes3.dex
[builder] INFO: Compiled classes4.dex
[builder] INFO: Compiled classes5.dex
[builder] INFO: Compiled classes6.dex
[builder] INFO: Compiled classes7.dex
[builder] INFO: Compiled classes8.dex
[builder] INFO: Compiling modified resources
```

### Device Environment

Galaxy A6+, OneUI 2.0 (Android 10).

### Acknowledgements

- [X] This issue does not reproduce on unpatched YouTube or YT Music.
- [X] This issue is not a duplicate of an existing bug report.
- [X] I have chosen an appropriate title.
- [X] All requested information has been provided properly.
- [X] I have written the title and contents in English.

**Comment by Raven-621** [comment]
> Does v6.42.54 work fine without any issues?
> 
> Asking if I should update.

Any version below 6.43 is fine.
Although Alireza (ReX) recently fixed this bug. So it's just until anddea updated the patch/integration that you should stay below 6.43

**Comment by anddea** [comment]
> Although Alireza (ReX) recently fixed this bug. So it's just until anddea updated the patch/integration that you should stay below 6.43

He didn't push his code for public visibility, otherwise it would be available here as well. But, nevermind, it's fixed in [v2.222.0-dev.8](https://github.com/anddea/revanced-patches/releases/tag/v2.222.0-dev.8). 

Too bad we have so many forks, we can't maintain just one together, instead each one of us writes his own fix. This way the project is going to be doomed one day.
